executor/executor_test.go

```diff
@@ -4077,7 +4075,7 @@ func (s *testSuite) TestShowTableRegion(c *C) {
 
 	// Test show table regions.
 	tk.MustExec(`set @@session.tidb_wait_split_region_finish=1;`)
-	tk.MustExec(`split table t_regions between (0) and (10000) regions 4;`)
+	tk.MustQuery(`split table t_regions by (2500),(5000),(7500);`).Check(testkit.Rows("3 1"))
```

Comment:
why changed the split stmt from `between ...` to `by (2500),(5000),(7500)` here? Is it possible to use this query with `between`?

Reply:
I just want to test more case. The `between ... and ...` syntax already test in line#4028.